go-controller/pkg/testing/mock_stubs.go

```diff
@@ -59,7 +60,32 @@ func (mock *MockOVNClient) LSLBList(ls string) ([]*goovn.LoadBalancer, error) {
 
 // Add ACL
 func (mock *MockOVNClient) ACLAdd(ls, direct, match, action string, priority int, external_ids map[string]string, logflag bool, meter string, severity string) (*goovn.OvnCommand, error) {
```

Comment:
Convention is to move the methods that will actually be supported by the mock to their own file. Like for instance this will move the erstwhile stubbed code to a file called mock_acl.go.

Reply:
OK, shall create a file called mock_acl.go